### PR TITLE
Fix symfony legacy route configuration, PART 2

### LINKED ISSUE #32306: Fix symfony legacy route configuration, PART 2
When using the command `php bin/console prestashop:linter:legacy-link`, these routes appear not to treated, it should be properly configure, it should be done, see [this issue](https://github.com/PrestaShop/PrestaShop/issues/31231) for more information.

- [x] admin_products_combinations_delete_combination
- [x] admin_products_combinations_bulk_delete
- [x] admin_products_attribute_groups
- [x] admin_all_attribute_groups
- [x] admin_products_combinations_generate
- [x] admin_products_specific_prices_list
- [x] admin_products_specific_prices_create
- [x] admin_products_specific_prices_edit
- [x] admin_products_specific_prices_delete
- [x] admin_products_v2_index
- [x] admin_products_v2_light_list
- [x] admin_products_search
- [x] admin_products_reset_grid_search
- [x] admin_products_grid_category_filter
- [x] admin_products_grid_shop_previews
- [x] admin_products_v2_download_virtual_product_file
- [x] admin_products_v2_delete_from_shop_group
- [x] admin_products_v2_delete_from_shop
- [x] admin_products_v2_enable_for_all_shops
- [x] admin_products_v2_enable_for_shop_group
- [x] admin_products_v2_disable_for_all_shops
- [x] admin_products_v2_disable_for_shop_group
- [x] admin_products_v2_search_associations
- [x] admin_products_v2_search_combinations
- [x] admin_products_v2_search_product_combinations
- [x] admin_products_v2_quantity
- [x] admin_categories_get_categories_tree
- [x] admin_monitorings_search
- [x] admin_monitoring_products_bulk_delete
- [x] admin_catalog_price_rules_list_for_product
- [x] admin_features_horizontal_index
- [x] admin_features_search
- [x] admin_features_export
- [x] admin_features_update_position
- [x] admin_attributes_index
- [x] admin_attributes_search
- [x] admin_attributes_create
- [x] admin_attributes_edit
- [x] admin_attributes_delete
- [x] admin_attributes_bulk_delete
- [x] admin_attributes_update_position
- [x] admin_cart_rules_create

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Checking and updating the whitelist of legacy links
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | -
| Fixed ticket?     | Fixes #{32306}
| Related PRs       | -
| Sponsor company   | -

### CODE REVIEW

File: src/PrestaShopBundle/Command/LegacyLinkLinterCommand.php [hunk 1]
```diff
@@ -57,6 +57,34 @@ class LegacyLinkLinterCommand extends Command
...
+        'admin_products_enable_for_shop_group',
+        'admin_products_disable_for_all_shops',
+        'admin_products_disable_for_shop_group',
+        'admin_products_search_associations',
+        'admin_products_search_combinations',
+        'admin_products_search_product_combinations',
+        'admin_products_quantity',
+        'admin_categories_get_categories_tree',
+        'admin_catalog_price_rules_list_for_product',
+        'admin_features_horizontal_index'
```
Comment (matthieu-rolland) [inline src/PrestaShopBundle/Command/LegacyLinkLinterCommand.php]:
```suggestion
        'admin_features_horizontal_index',
```

CS fixer is complaining about this missing comma :+1:

**Review by matthieu-rolland — COMMENTED** [review]
LGTM, nice job :+1:

File: src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products_v2/product.yml [hunk 2]
```diff
@@ -331,7 +332,7 @@ admin_products_bulk_disable_shop_group:
     _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\Product\ProductController::bulkDisableShopGroupAction
     _legacy_controller: AdminProducts
     _legacy_link:
-      - AdminProducts:submitBulkenableSelectionproduct
+      - AdminProducts:submitBulkdisableSelectionproduct
```
Comment (boherm) [inline src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products_v2/product.yml]: Maybe we can simplify this:
`_legacy_link: AdminProducts:submitBulkdisableSelectionproduct`

**Review by boherm — CHANGES_REQUESTED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
QA dev ok ! treated routes are not appearing anymore when executing the command 👍

**Pull request merged**